library/alloc/src/vec.rs

```diff
@@ -567,9 +567,10 @@ impl<T> Vec<T> {
         self.buf.try_reserve(self.len, additional)
     }
 
-    /// Tries to reserves the minimum capacity for exactly `additional` more elements to
-    /// be inserted in the given `Vec<T>`. After calling `try_reserve_exact`,
-    /// capacity will be greater than or equal to `self.len() + additional`.
+    /// Tries to reserve the minimum capacity for exactly `additional` more
+    /// elements to be inserted in the given `Vec<T>`. After calling
+    /// `try_reserve_exact`, capacity will be greater than or equal to
```

Comment:
This seems wrong -- presumably it means "will be greater than... if returning Ok(())"? Or something like that...

Reply:
Err, I just rearrange the text, I didn't change anything besides `s/to reserves/to reserve/`. Or maybe should I change it here?

Reply:
Wait, it could be equal to right? Is it wrong? ~~But IIRC it just does additional work if the capacity and the `additional` is the same.~~ additional work was `truncate` and `resize`, not this one.

Reply:
I would prefer to update it here. If you're going to be fixing doc comments, I would prefer that we indeed put in the effort to fix them fully; just fixing a single typo but leaving other bugs seems insufficient to me.

Reply:
Are you sure it means "will be greater than"? I thought it mean it could be "equal to" too? I don't see why it is wrong.

Reply:
I am specifically referring to the fact that this function *can fail*, not the greater than or equal to part. That's why I included the "..."

Reply:
Oh, haha I thought you mean something else. Sorry.